CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+
+The project maintainer will create a [GitHub Discussion](github.com/operator-framework/operator-controller/discussions) for the upcoming milestone once we've finalized the current milestone. Please feel encouraged to contribute suggestions for the milestone in the discussion.
+
+## Where are Operator Controller Milestones?
+
+Ongoing or previous Operator-Controller milestones can always be found in the [milestone section of our GitHub Repo](https://github.com/operator-framework/operator-controller/milestones).
+
+### How are Subproject Issues Tracked?
+
+As discussed earlier, the operator-controller adheres to a microservice architecture, where multiple projects contribute to the overall experience. As such, when designing an operator-controller milestone, the community may need to file an issue against RukPak, Deppy, or Catalogd. Unfortunately, the operator-controller milestone cannot contain issues from one of its subprojects. As such, we've introduced the concept of a "Dependency Issue", described below:
+
+> Dependency Issues: An issue tracked in a milestone that "points" to an issue in another project with a URL.
+
+## Submitting Issues
+
+Unsure where to submit an issue? 
+- The Operator-Controller project
+- The RukPak project
+- The Deppy project
+- The Catalogd project
+
+Don't worry if you accidentally submit an issue against the wrong project, if we notice that an issue would fit better with a separate project we'll move it to the correct repository and mention it in the #olm-dev slack channel.
+
+## Submitting Pull Requests
+
+### Code Review
+
+Contributing PRs with a reasonable title and description can go a long way with helping the PR through the review
+process.
+
+When opening PRs that are in a rough draft or WIP state, prefix the PR description with `WIP: ...` or create a draft PR.
+This can help save reviewer's time by communicating the state of a PR ahead of time. Draft/WIP PRs can be a good way to
+get early feedback from reviewers on the implementation, focusing less on smaller details, and more on the general
+approach of changes.
+
+When contributing changes that require a new dependency, check whether it's feasible to directly vendor that
+code [without introducing a new dependency](https://go-proverbs.github.io/).
+
+Currently, PRs require at least one approval from a operator-controller maintainer in order to get merged.
+
+### Code style
+
+The coding style suggested by the Golang community is used throughout the operator-controller project:
+
+- [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments)
+- [EffectiveGo](https://golang.org/doc/effective_go)
+
+In addition to the linked style documentation, operator-controller formats Golang packages using the `golangci-lint` tool. Before
+submitting a PR, please run `make lint` locally and commit the results. This will help expedite the review process,
```

Comment:
I don't think this make target is actually in the project yet :sweat_smile:

Reply:
Oh, let's add that as a follow up issue.